main/acle.md

```diff
@@ -4855,7 +4885,17 @@ one of the following is true:
 
 *   F shares ZT0 with its caller
 
-All other functions have a private-ZA interface.
+All other functions have either a private-ZA or an agnostic-ZA interface.
+
+If F implements an agnostic-ZA interface and PSTATE.ZA is available at runtime,
+then a call to F must return with its ZA state unchanged in accordance
+to the [[AAPCS64]](#AAPCS64). In practice this means that calls to F don't have
+to emit code to set up a lazy-save for ZA or to preserve other state like ZT0
+when such state is live at the call site.
+
+The implementation of F must not make any assumptions on the availability of
```

Comment:
I think we should instead add a note to the attribute documentation to say that it does not imply that SME is available.